docs/wiki/01_STUDY_PARAMETERS_SETUP/04_Data_Querying_Tool/README.md

```diff
@@ -0,0 +1,85 @@
+# Data Query Tool
+
+## Purpose
+
+The Data Query Tool (“The DQT”) is used to query and export data stored in LORIS. It is located within the Reports drop-down menu. 
+
+A variety of filters allow users to define parameters and query the exact data fields desired. The extracted data can be downloaded as a single CSV file, and imaging files can also be downloaded as a zipped package of files. Users can visualize or see basic statistical measures from a query. Users can load a previously saved query, and can make saved queries visible to other users.
+
+## Intended Users
+
+The DQT module is intended for researchers who wish to download data stored in LORIS. Users can access only internal data that belongs to their LORIS instance. 
+
+## Scope
+
+The module implements a proxy to pass through queries to the Loris CouchDB DQT backend, which must be setup independently (see Configuration section).
+
+It does NOT alter data in any way other than that required to convert the data from the MySQL to the CouchDB format. In particular, it does no anonymization or filtering of participant data that is already in LORIS. This must be done independently, before distributing data externally.
+
+## Other Notes
+
+The most closely related module to the DQT is the Data Release module, which is used to store snapshots of data for any given release. 
+
+## Permissions
+
+Users must have `dataquery_view` permission to use the DQT. 
+
+It’s important to note that granular permissions do not exist for this module. Anyone with the `dataquery_view` permission has access to view, query and download data from the DQT, and therefore has access to all data loaded in the DQT from this LORIS instance. It is not possible to restrict user access to a subset of the data.
+
+## Interactions with LORIS
+
+Before the DQT can be used, the CouchDB import scripts must be run in order to load data from LORIS's MySQL tables into CouchDB. These scripts are found in the `tools/` directory, named `CouchDB_Import_*.php`.
+
+It is recommended to set up a cronjob to execute these scripts to refresh the data on a regular basis. 
+
+## Configuration
+
+Use of the DQT module requires setting up the CouchDB server and credentials:
+
+[CouchDB](http://couchdb.apache.org)
+
+Once CouchDB is installed, follow these steps to complete setup:
+
+1. Create a database on your local CouchDB instance
+
+2. Clone the code from our server:
+
+`
+curl -H 'Content-Type: application/json' -X POST http://$YOURCOUCHDBADMIN:$YOURCOUCHADMINPASS@$YOURSERVERNAME:5984/_replicate -d '{"source":"http://couchdb.loris.ca:5984/dataquerytool-1_0_0", "target":"$YOURDATABASENAME"}'
```

Comment:
This section does not display well (list numerotation issue and code block vertical spacing). See my comment to fix it [here](https://github.com/aces/Loris/issues/7080#issuecomment-702939618)